tests/config_macsec_test.py

```diff
@@ -0,0 +1,130 @@
+from unittest import mock
+
+from utilities_common.db import Db
+
+from click.testing import CliRunner
+import config.main as config
+
+
+def test_macsec_default_profile():
+    runner = CliRunner()
+    db = Db()
+
+    profile_name = "test"
+    primary_cak = "01234567890123456789012345678912"
+    primary_ckn = "01234567890123456789012345678912"
+    result = runner.invoke(config.config.commands["macsec"].commands["profile"].commands["add"], [profile_name, "--primary_cak=" + primary_cak,"--primary_ckn=" + primary_ckn], obj=db)
+    assert result.exit_code == 0, "exit code: {}, Exception: {}, Traceback: {}".format(result.exit_code, result.exception, result.exc_info)
+    profile_table = db.cfgdb.get_entry("MACSEC_PROFILE", profile_name)
+    assert profile_table
+    assert profile_table["priority"] == "255"
+    assert profile_table["cipher_suite"] == "GCM-AES-128"
+    assert profile_table["primary_cak"] == primary_cak
+    assert profile_table["primary_ckn"] == primary_ckn
+    assert profile_table["policy"] == "security"
+    assert "enable_replay_protect" not in profile_table
+    assert "replay_window" not in profile_table
+    assert profile_table["send_sci"] == "1"
+    assert "replay_period" not in profile_table
```

Comment:
I don't think there is any field called replay_period?  There is only replay_window and rekey_period AFAIK?